src/server/services/wms/qgswmsgetcapabilities.cpp

```diff
@@ -54,30 +53,17 @@ namespace QgsWms
     void appendDrawingOrder( QDomDocument &doc, QDomElement &parentElem, QgsServerInterface *serverIface,
                              const QgsProject *project );
 
-    void combineExtentAndCrsOfGroupChildren( QDomDocument &doc, QDomElement &groupElem, const QgsProject *project,
-        bool considerMapExtent = false );
-
-    bool crsSetFromLayerElement( const QDomElement &layerElement, QSet<QString> &crsSet );
-
-    QgsRectangle layerBoundingBoxInProjectCrs( const QDomDocument &doc, const QDomElement &layerElem,
-        const QgsProject *project );
+    void appendLayerWgs84BoundingRect( QDomDocument &doc, QDomElement &layerElement, const QgsRectangle &wgs84BoundingRect );
 
-    void appendLayerBoundingBox( QDomDocument &doc, QDomElement &layerElem, const QgsRectangle &layerExtent,
-                                 const QgsCoordinateReferenceSystem &layerCRS, const QString &crsText,
-                                 const QgsProject *project );
-
-    void appendLayerBoundingBoxes( QDomDocument &doc, QDomElement &layerElem, const QgsRectangle &lExtent,
-                                   const QgsCoordinateReferenceSystem &layerCRS, const QStringList &crsList,
-                                   const QStringList &constrainedCrsList, const QgsProject *project,
-                                   const QgsRectangle &geoExtent = QgsRectangle() );
+    void appendLayerCrsExtents( QDomDocument &doc, QDomElement &layerElement, const QMap<QString, QgsRectangle> crsExtents );
 
     void appendCrsElementToLayer( QDomDocument &doc, QDomElement &layerElement, const QDomElement &precedingElement,
                                   const QString &crsText );
 
     void appendCrsElementsToLayer( QDomDocument &doc, QDomElement &layerElement,
                                    const QStringList &crsList, const QStringList &constrainedCrsList );
 
-    void appendLayerStyles( QDomDocument &doc, QDomElement &layerElem, QgsMapLayer *currentLayer,
+    void appendLayerStyles( QDomDocument &doc, QDomElement &layerElem, const layerInfos &linfos,
```

Comment:
```suggestion
    void appendLayerStyles( QDomDocument &doc, QDomElement &layerElem, const LayerInfos &layerInfos,
```

Sorry but `linfos` makes me think of  bad body illnes  :)

Reply:
With the changes in name struct I can reused layerInfos.